autotest/gcore/rasterio.py

```diff
@@ -845,6 +845,34 @@ def test_rasterio_16():
 
 ###############################################################################
 
+def test_rasterio_nodata():
+
+	try:
+		from osgeo import gdalnumeric
+		gdalnumeric.zeros
+		import numpy
```

Comment:
you should remove this 'import numpy' as it is not use in your function.
But the indentation of this block and the rest of your code is not consistent